NaviGator/mission_control/navigator_missions/navigator_missions/scan_the_code.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+import asyncio
+from operator import attrgetter
+
+import numpy as np
+import txros
+from cv2 import bitwise_and
+from cv_bridge import CvBridge
+from dynamic_reconfigure.msg import DoubleParameter
+from image_geometry import PinholeCameraModel
+from mil_tools import numpy_to_pointcloud2 as np2pc2
+from mil_tools import rosmsg_to_numpy
+from mil_vision_tools.cv_tools import contour_mask, rect_from_roi, roi_enclosing_points
+from navigator_msgs.msg import ScanTheCode
+from sensor_msgs.msg import Image, PointCloud2
+from std_srvs.srv import SetBoolRequest
+
+from .navigator import Navigator
+
+LED_PANEL_MAX = 0.1
```

Comment:
Units? We should add this as a comment following the number like
```python
CONSTANT_WITH_UNITS = 0.6 # meters
```

Reply:
agreed